docs/src/content/docs/getting-started/getting-started.md

```diff
@@ -445,8 +445,6 @@ async function runApp() {
     to: address,
     value: 420n,
     throwOnFail: false,
-      // on-success will only create a transaction if the initial run of it doesn't revert
```

Comment:
I’m assuming this was supposed to be missing from the first code block given the next section adds it

Reply:
ah good catch